src/main/java/com/github/nicholasmoser/gnt4/seq/ext/SeqAssembler.java

```diff
@@ -0,0 +1,1304 @@
+package com.github.nicholasmoser.gnt4.seq.ext;
```

Comment:
In IntelliJ, along the right side of the class, there are some suggestions made to help clean up this class:

![image](https://user-images.githubusercontent.com/4983605/174518780-12873d28-c258-4a03-ae1d-30d909530a4e.png)

If you hover over the yellow bars in the above image IntelliJ makes recommendations. You can also view these under the Problems view:

![image](https://user-images.githubusercontent.com/4983605/174518893-48689fe8-4683-4ae2-b578-6464e3eb9029.png)

For the best recommendations, make sure the language level of your project is set to 17 under `File -> Project Structure...`:

![image](https://user-images.githubusercontent.com/4983605/174518977-a560d2ca-d5e6-4a46-89d0-71e5753226a4.png)